### PR TITLE
doc: add minutes for meeting 2017-03-13

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by ss74** [comment]
Good

File: doc/meetings/2017-03-13.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+* Backporting WG [#184](https://github.com/nodejs/LTS/issues/184)
+
+## Minutes
+
+### Potential Semver Minor Backports [#177](https://github.com/nodejs/LTS/issues/177)
+  * These have already landed, minor went out 2 weeks ago
+  * Need to create newer list, to review next meeting.
+
+### Should we consider swapping the timeframe for Active and Maintenance [#130 ](https://github.com/nodejs/LTS/issues/130)
+* Myles: seems like longer Active was useful, so he.s ok
```
Comment (gibfahn) [inline doc/meetings/2017-03-13.md]: `he.s`->`he's`

File: doc/meetings/2017-03-13.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  with keeping as it for now
+* James: have been some calls to have longer LTS period (5 period),
```
Comment (gibfahn) [inline doc/meetings/2017-03-13.md]: `5 period` -> `5 year period`

File: doc/meetings/2017-03-13.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  this is a request for a longer overall LTS period as opposed to
+  changing the split.  But probably points to us not wanting to
+  shorten the active period.
+* Jeremiah - suggested we might be able to let more changes into Maintenance.
+  If no request will just do critical/security fixes.  Could allow other
+  changes on request at the discretion of the LTS team.
+* James/Michael - we could allow on request.  Maintenance, we
+  do not it actively but if somebody wanted to backport we could
```
Comment (gibfahn) [inline doc/meetings/2017-03-13.md]: `do not` -> `do not do`

File: doc/meetings/2017-03-13.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+Next actions:
+* leave Active/Maintenance as is for now
+* open new issue to better define release schedule, this might
+  include release 1/quarter once in Maintenance.  For scheduled minors,
+  might just be were it would be if the minor is needed.  Myles will
+  take action to open issue, it will include proposed release schedule template.
+
+### Managing floating V8 patches [#111]https://github.com/nodejs/LTS/issues/111)
+* Myles: This has been pretty much under control.  Ali has created good
```
Comment (gibfahn) [inline doc/meetings/2017-03-13.md]: `Ali`->`Ali (@ofrobots)`

File: doc/meetings/2017-03-13.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+* Michael Dawson (@mhdawson)
+* Myles Borins 
+* James Snell (@jasnell)
+* Jeremiah Senkpiel (@fishrock123)
+* Gibson Fahnestock (@gibfan)
+
+## Agenda
+* Potential Semver Minor Backports [#177](https://github.com/nodejs/LTS/issues/177)
+* Should we consider swapping the timeframe for Active LTS and
+  Mainteance [#130 ](https://github.com/nodejs/LTS/issues/130)
```
Comment (sam-github) [inline doc/meetings/2017-03-13.md]: "maintenance"

File: doc/meetings/2017-03-13.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+  If no request will just do critical/security fixes.  Could allow other
+  changes on request at the discretion of the LTS team.
+* James/Michael - we could allow on request.  Maintenance, we
+  do not it actively but if somebody wanted to backport we could
+  land, not necessarily create a release. Would be limited to patch.
+* Michael ok, but limited to releases that are not EOL.
+  Myles should not land unless we are going to do a release.
+* If we do takes more changes into Maintenance phase, when will
+  builds actually go out.  Mostly in the past those have been
+  for security releases, and we don.t include any other
```
Comment (sam-github) [inline doc/meetings/2017-03-13.md]: "don't"

File: doc/meetings/2017-03-13.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+  changes with those, so when would the changes go in ? 
+* Myles, possibly if in Maintenance mode, if we accept PRs
+  outside the critical ones, then we should have a release once/quarter.
+
+Next actions:
+* leave Active/Maintenance as is for now
+* open new issue to better define release schedule, this might
+  include release 1/quarter once in Maintenance.  For scheduled minors,
+  might just be were it would be if the minor is needed.  Myles will
```
Comment (sam-github) [inline doc/meetings/2017-03-13.md]: "were"->"where"?

File: doc/meetings/2017-03-13.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+  take action to open issue, it will include proposed release schedule template.
+
+### Managing floating V8 patches [#111]https://github.com/nodejs/LTS/issues/111)
+* Myles: This has been pretty much under control.  Ali has created good
+  backporting guide.  What was not resolved in this issue was how we manage v8.
+  We had discussed having nodejs/v8 tree, but we are still just floating in
+  the node repo. https://github.com/nodejs/v8 exists but Myles does not
+  believe we are using as a staging group.  Seems ok now as
+  we don.t need an extra stage of indirection until we have tooling
```
Comment (sam-github) [inline doc/meetings/2017-03-13.md]: "don't"

File: doc/meetings/2017-03-13.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+* Michael, is that document useful for when we ask people to do backport
+  because it did not land cleanly.
+* Jeremiah, doc was supposed to cover all release branches, but current
+  versus LTS releases may be different.
+* Myles, process really should be the same, same meta data etc.
+* Generally we are agreed, that a team for backporting would make sense. 
+
+Next actions: 
+  * Volunteers,   James, Gibson, Myles, Sam  (10 hours each ?)
+  * Myles will send out email to arrange time.
```
Comment (sam-github) [inline doc/meetings/2017-03-13.md]: I can spend a few days a month, no problem. We'll have another meeting?

**Comment by mhdawson** [comment]
Addressed first set of comments @gibfahn @sam-github please take another look.

**Comment by sam-github** [comment]
LGTM, but I wasn't at  the meeting, I think the folks who were need to LGTM that it reflects what happened.

**Comment by mhdawson** [comment]
@jasnell could you do a quick review ?

**Comment by mhdawson** [comment]
Landed  as bbec3c651cfeb73596114ac367791ae5a38666c3

**Pull request closed without merging**